nestkernel/archiving_node.cpp

```diff
@@ -275,6 +275,19 @@ nest::Archiving_Node::set_status( const DictionaryDatum& d )
     clear_history();
   }
 
+
+  if ( d->known( names::update_synaptic_elements ) )
```

Comment:
Is update_synaptic_elements a good name? It represents the value to update the element with, and not a boolean. Hmm, I'm not inspired as to what would be a better short word for update which doesn't sound like a true/false but is the value to update with...